### PR TITLE
Modify the block template download method to pnpm, modify the block release environment variables

### PR DESCRIPTION
**Pull request by lu-yg**
English | [简体中文](https://github.com/opentiny/tiny-engine-webservice/blob/main/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine-webservice/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Updated npm configuration commands for improved authentication and registry settings.
	- Introduced a new property for domestic npm mirrors in the configuration settings.
	- Shifted package management from npm to pnpm in the VueBlockBuilder class.

- **Bug Fixes**
	- Corrected the registry URL format in the configuration for better compatibility. 

- **Style**
	- Improved formatting consistency in the VueBlockBuilder class.

### CODE REVIEW

File: config/config.default.ts [hunk 1]
```diff
@@ -296,10 +296,14 @@ export default (appInfo) => {
...
 
+  config.cnpmRegistryOptions = [
+    '--registry=http://registry.npmmirror.com/'
+  ];
   config.buildground = '/tmp/buildground';
   config.baseNpm = '@opentiny/tiny-engine-block-build';
   config.authToken = process.env.NPM_AUTH_TOKEN; // 替换为自己的npm token
-  config.registry = 'registry.npmjs.org/';
+  config.registry = 'https://registry.npmjs.org/';
+  config.tokenRegistry = 'registry.npmjs.org/';
```
Comment (wenmine) [inline config/config.default.ts]: 这里建议改成：config.tokenRegistry = config.registry.split("//")?.[1]

File: config/config.default.ts [hunk 2]
```diff
@@ -296,10 +296,14 @@ export default (appInfo) => {
...
   ];
 
+  config.cnpmRegistryOptions = [
+    '--registry=http://registry.npmmirror.com/'
+  ];
   config.buildground = '/tmp/buildground';
   config.baseNpm = '@opentiny/tiny-engine-block-build';
   config.authToken = process.env.NPM_AUTH_TOKEN; // 替换为自己的npm token
-  config.registry = 'registry.npmjs.org/';
+  config.registry = 'https://registry.npmjs.org/';
```
Comment (wenmine) [inline config/config.default.ts]: 让用户改这里的话，会不会影响的是整个源？比如说他在这里会不会结合自己搭建的私仓源+公司内部其他源使用？还是说这里配置的源就是给区块发布专用的？

File: config/config.default.ts [hunk 3]
```diff
@@ -296,10 +296,14 @@ export default (appInfo) => {
     '--registry=https://registry.npmjs.org/',
   ];
 
+  config.cnpmRegistryOptions = [
+    '--registry=http://registry.npmmirror.com/'
```
Comment (wenmine) [inline config/config.default.ts:313]: 这里加了一个新配置参数，在这里注释一下这个新配置参数的用途吧，还要检查一下是否需要调整我们原来的文档

File: config/config.default.ts (hunk 2 above)
Comment (lu-yg) [reply config/config.default.ts]: 只给区块用的

**Review by wenmine — APPROVED** [review]

**Pull request closed without merging**